### PR TITLE
Remove unnecessary prepare work

### PR DESCRIPTION
**Pull request by andrewgordstewart**

### CODE REVIEW

File: packages/channel-provider/package.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
     "prepublish": "webpack",
     "build": "webpack",
     "build:typescript": "tsc -b .",
-    "prepare": "yarn build",
```
Comment (kerzhner) [inline packages/channel-provider/package.json]: `channel-client` and `embedded-wallet` import `channel-provider`. `eslint` will fail during the circle `build` job.

File: packages/rps/package.json [hunk 2]
```diff
@@ -9,11 +9,10 @@
     "firebase/**/grpc": "^1.16.1"
   },
   "scripts": {
-    "build:ci": "run-s clearContracts build:typescript contracts:compile",
+    "build:ci": "run-s build:typescript contracts:compile",
```
Comment (kerzhner) [inline packages/rps/package.json]: Do contracts need to be compiled? Contracts are already compiled (on every circle run) in `prepare` job.

File: packages/channel-provider/package.json (hunk 1 above)
Comment (kerzhner) [reply packages/channel-provider/package.json]: The prepare command should probably be switched to run `yarn build:typescript`

File: packages/rps/package.json (hunk 2 above)
Comment (andrewgordstewart) [reply packages/rps/package.json]: Good question. On the one hand, it's nice for `build:ci` to do the complete build; the whole _point_ of `build:ci` is to check that the package can be built from start to finish. On the other hand, the current `build:ci` is a bit strange: contracts are deleted, then the typescript is compiled, then contracts are compiled, and webpack is _not_ run.

For now, perhaps I'll remove `contracts:compile` from `build:ci`

**Pull request merged**